src/components/Change/index.js

```diff
@@ -0,0 +1,154 @@
+import React, { Component } from 'react'
+import { graphql } from 'gatsby'
+import styled from 'styled-components'
+import Panel from 'components/Panel'
+import Markdown from 'components/Markdown'
+import Link from 'components/Link'
+import { grayscale } from 'utils/colors'
+import { weight } from 'utils/fonts'
+
+const markdownComponents = {
+  h1({ children, ...props }) {
+    return <h4 {...props}>{children}</h4>
+  },
+  h2({ children, ...props }) {
+    return <h5 {...props}>{children}</h5>
+  },
+  h3({ children, ...props }) {
+    return <h5 {...props}>{children}</h5>
```

Comment:
pls explain `h2` & `h3` = `h5` in chat

Reply:
We don't want any titles larger than `h4` in the change contents.

So if we have any, we shrink them down to `h4` or `h5`. But we also want them to change relative to their original size.

So we shift `h1` to `h4` and `h2`/`h3` to `h5`.